lib/Core/Executor.cpp

```diff
@@ -2998,12 +2977,33 @@ void Executor::run(ExecutionState &initialState) {
     }
 
     klee_message("seeding done (%d states remain)", (int) states.size());
+    klee_message("<CONC> sizeof prio2: %lu", prio2.size());
+    klee_message("<CONC> sizeof prio1: %lu", prio1.size());
+    klee_message("<CONC> sizeof prio0: %lu", prio0.size());
+    for ( auto &s : prio2 ) {
+        //klee_message("Solving prio2");
+        interpreterHandler->processTestCase(*s, 0, 0); // TODO: maybe create our own terminate function
+        interpreterHandler->incPathsExplored(); // not sure this is correct
+    }
+    for ( auto &s : prio1 ) {
+        interpreterHandler->processTestCase(*s, 0, 0); // TODO: maybe create our own terminate function
+        interpreterHandler->incPathsExplored(); // not sure this is correct
+    }
+    /*for ( auto &s : prio0 ) {
+        klee_message("Solving prio0");
+        interpreterHandler->processTestCase(*s, 0, 0); // TODO: maybe create our own terminate function
+        interpreterHandler->incPathsExplored(); // not sure this is correct
+    }*/
+
+    klee_message("states size() after: %lu\n", states.size());
 
     if (OnlySeed) {
       doDumpStates();
```

Comment:
I wonder if this call will duplicate some of the dumping we do above. In this line [R1050](https://github.com/leonbett/klee/pull/1/commits/56889ff49281fc21d826246fca244487be420232#diff-1b304792b1e06d7ee52b0f6fe724e9cbR1050), we're adding to our prio list but we're not killing the state, which makes me think KLEE will have a state that will be dumped above and in the doDumpStates() call.